examples/04_glm_first_level/plot_write_events_file.py

```diff
@@ -85,17 +240,19 @@
 #########################################################################
 # Export them to a tsv file.
 import os
-outdir = 'results'
+
+outdir = "results"
 if not os.path.exists(outdir):
     os.mkdir(outdir)
-tsvfile = outdir + '/localizer_events.tsv'
-events.to_csv(tsvfile, sep='\t', index=False)
-print(f'The event information has been saved to {tsvfile}')
+tsvfile = f"{outdir}/localizer_events.tsv"
```

Comment:
I would rely on pathlib for this kind of path manipulations. Or at least use `os.path.join`.

Reply:
done

I suspect that in another PR we could try to use Path everywhere possible in the examples.